swift-sdk/Internal/InternalIterableAPI.swift

```diff
@@ -471,6 +451,27 @@ final class InternalIterableAPI: NSObject, PushTrackerProtocol, AuthProvider {
         try? requestHandler.handleLogout()
     }
     
+    private func storeIdentifierData() {
+        localStorage.email = _email
+        localStorage.userId = _userId
+    }
+    
+    private func postIdentifierActions() {
```

Comment:
I think it's not obvious what this method does from its name without looking at its code... At first I thought "it seems to.. post identifier actions? what does that mean?".

Reply:
You're right. It kind of sounds like a line from a sci-fi movie. I've renamed it to `postLoginActions`.

Reply:
Oh, I also meant that since most method names are usually verbs, and `post` is an ambiguous word, I start thinking this thing posts login actions... somewhere.

Reply:
Ohhhh! Okay, changed to `onLogin` like on Android. That's clearer.